test/jdk/javax/net/ssl/HttpsURLConnection/CriticalSubjectAltName.java

```diff
@@ -27,31 +27,60 @@
 //
 
 /*
- * @test
+ * @test id=tls12
  * @bug 6668231
  * @summary Presence of a critical subjectAltName causes JSSE's SunX509 to
  *          fail trusted checks
- * @run main/othervm CriticalSubjectAltName
+ * @library /test/lib
+ * @modules java.base/sun.security.x509 java.base/sun.security.util
+ * @run main/othervm CriticalSubjectAltName TLSv1.2 MD5withRSA
  * @author Xuelei Fan
  */
 
+
 /*
- * This test depends on binary keystore, crisubn.jks and trusted.jks. Because
```

Comment:
Peease remove the crisubn.jks and trusted.jks files from this directory, they aren't used anywhere else.